### PR TITLE
Fix images issues on HDPI resolution

### PR DESCRIPTION
**Pull request by octopusSD**
This fixes some issue related to HDPI resolution (4k in my case) where all icons looks very "pixelated". Not tested on non HDPI resolution.
Screenshot_20191225_172323

### CODE REVIEW

**Comment by vinifmor** [comment]
Great, @octopusSD !  I'm going to test your changes against no HDPI devices ( I don't have any HDPI device at the moment ) and merge them as soon as possible. Thank you for helping the project.

File: bauh/view/qt/window.py [hunk 1]
```diff
@@ -777,7 +778,7 @@ def _update_type_filters(self, available_types: dict = None, keep_selected: bool
                     icon = self.cache_type_filter_icons.get(app_type)
 
                     if not icon:
-                        icon = load_icon(icon_path, 14)
+                        icon = load_icon(icon_path, 18)
```
Comment (vinifmor) [inline bauh/view/qt/window.py]: Hi @octopusSD. I've been testing the application with your changes, and lines 161 and 781 make the icons bigger for non-hdpi resolutions. Do these new sizes make a big difference for HDPI resolutions ? ( I'm currently with no HDPI device to test, that's why I'm asking ). The pictures below show how non-hdpi devices are affected:

**Without changes**
no_pr

**With changes**
pr_1

**Comment by octopusSD** [comment]
I think it will be better if just remove icon from default Type item in filter by type combo box for more consistency look with category combo box. I also reverted the size of package icon. Now it looks like this on my 4k display:
Screenshot_20200108_211830

**Comment by vinifmor** [comment]
The icons look pixelated in your screen. I will try to replace them by svgs to see if it fixes this issue.

**Comment by vinifmor** [comment]
I merged your PR in a separate branch (**svg**) for the moment so we can try to address this issue together. I replaced all the png icons by svgs. Let me know if that helps.

**Comment by octopusSD** [comment]
No it still looks pixelated. But I found another workaround. Turns out using QPixmap to load the svg (or any other image type) and then scaling it up with pixmap.scaled(width, height) will cause the end result to be pixelated. Passing it in directly as QIcon(image_path) will scale it properly. And if pixmap needed it can be scaled from QIcon example icon.pixmap(QSize(width, height)).

**Comment by octopusSD** [comment]
Now it looks like this:
Screenshot_20200109_125334

**Comment by vinifmor** [comment]
It seems better for me, but I'm not a HDPI user haha What do you think ?

**Comment by octopusSD** [comment]
It looks much better now from what it was in the beginning, everything is sharp and clean. I think you can merge this PR if it is OK from your side (and resolution).

**Comment by vinifmor** [comment]
Great. I'm going to release a RC in the end of the day with these changes. It will be available in AUR (**bauh-staging)**. Thank you for helping the project.

**Comment by vinifmor** [comment]
Already available in **AUR** as **bauh-staging** (0.8.1.RC-3)

**Pull request merged**